requirements.txt

```diff
@@ -14,7 +14,7 @@ passlib==1.6.2
 python-dateutil==2.4.2
 python-mimeparse==0.1.4
 six==1.10.0
-git+git://github.com/xcompass/Flask-CAS.git
+git+git://github.com/cameronbwhite/Flask-CAS.git@v1.0.1
```

Comment:
If it works, please feel free to merge the PR. And let me know so I can deprecate my forked repo.